src/averageAges.js

```diff
@@ -15,12 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+  const men = century
+    ? people.filter((gender) => gender.sex === 'm'
+      && Math.ceil(gender.died / 100) === century)
+    : people.filter((gender) => gender.sex === 'm');
+
+  const age = men.map((years) => years.died - years.born);
```

Comment:
```suggestion
  const age = men.map((man) => man.died - man.born);
```
It's not years. You iterate through men